File: Beecrowd/bc_python/hello_bee/bc_1049_animal.py
def main():

  carac_1  = 'invertebrado'
  carac_2  = 'anelideo'
  carac_3  = 'hematofago'
  # carac_1  = input().split(' ')
  # carac_2  = input().split(' ')
  # carac_3  = input().split(' ')

  definir_nome_do_animal(carac_1, carac_2, carac_3)
  


def definir_nome_do_animal(carac_1, carac_2, carac_3):
    if carac_1 == 'vertebrado':
        if carac_2 == 'ave':
            if carac_3 == 'carnivoro':
                print('aguia')
                
            elif carac_3 == 'onivoro':
                print('pomba')
                
        elif carac_2 == 'mamifero':
            if carac_3 == 'onivoro':
                print('homem')
            elif carac_3 == 'herbivoro':
                print('vaca')
      
    elif carac_1 == 'invertebrado':
        if carac_2 == 'inseto':
            if carac_3 == 'hematofago':
                print('pulga')
            else:
                print('lagarta')
        else:
            if carac_3 == 'hematofago':
                print('sanguessuga')
            else:
                print('minhoca')

main()

File: Beecrowd/bc_python/hello_bee/bc_1049_animal.js
import { import_files } from '../my_io_utils.js'

function main(){

  const lines = import_files('teste/bc_1049.txt')
  
  
  const carac_1  = lines[0]
  const carac_2  = lines[1]
  const carac_3  = lines[2]

  // console.log(carac_1, carac_2, carac_3)

  definir_nome_do_animal(carac_1, carac_2, carac_3)
  
}

function definir_nome_do_animal(carac_1, carac_2, carac_3){
/*     if (carac_1 == 'vertebrado'){
        if (carac_2 == 'ave'){
            if (carac_3 == 'carnivoro'){
                console.log('aguia')
                
            } else {
                console.log('pomba')
            }
                
        } else { 
            if( carac_3 == 'onivoro'){
                console.log('homem')
            } else { 
                console.log('vaca')
            }
        }
      
    } else {
        if( carac_2 == 'inseto'){
            if( carac_3 == 'hematofago'){
                console.log('pulga')
            } else {
                console.log('lagarta')
            }
        } else {
            if (carac_3 == 'hematofago'){
                console.log('sanguessuga')
            } else {
                console.log('minhoca')
            }
        }
    }
} */

  if (carac_1 == 'vertebrado'){
    if (carac_2 == 'ave') {
      if (carac_3 == 'carnivoro'){
        console.log('aguia')
        
      } else {
        console.log('pomba')
        
      }
    } else {
      if ( carac_3 == 'onivoro') {
        console.log('homem')
      } else {
         console.log('vaca')
      }
    }
  } else {
    if (carac_2 == 'inseto'){
      if( carac_3 == 'hematofago'){
        console.log('pulga')
      } else {
        console.log('lagarta')
      }
    } else {
      if (carac_3 == 'hematofago'){
        console.log('sanguessuga')
      } else {
        console.log('minhoca')
      }
    }
  }
}

main()